WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -3,20 +3,32 @@ import Foundation
 class ReaderCardsStreamViewController: ReaderStreamViewController {
     private var currentPage = 1
 
+    private let readerCardTopicsIdentifier = "ReaderTopicsCell"
+
     lazy var cardsService: ReaderCardService = {
         return ReaderCardService()
     }()
 
     override func viewDidLoad() {
         super.viewDidLoad()
         ReaderWelcomeBanner.displayIfNeeded(in: tableView)
+        tableView.register(ReaderSuggestedTopicsCell.self, forCellReuseIdentifier: readerCardTopicsIdentifier)
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        // Only sync if the tableview is at the top, otherwise this will change offset
+        if isTableViewAtTheTop() {
+            super.viewWillAppear(animated)
```

Comment:
Not sure this is "allowed" from the documentation:
`If you override this method, you must call super at some point in your implementation`
Maybe you always call `super.viewWillAppear(animated)` but do:  
 if isTableViewAtTheTop() {
  tableView.reloadData() 
}
?

Reply:
Hmm, I wasn't sure about that.

I'll figure out another solution.

Reply:
Changed in 9f28f40c2288a5a5f79803dda39dab3143b15ca4